web/controllers/weekly_info_controller.ex

```diff
@@ -3,17 +3,31 @@ defmodule Syndeo.WeeklyInfoController do
   alias Syndeo.WeeklyInfo
   alias Syndeo.Attendee
   import Syndeo.AttendeeQuery
+  import Syndeo.MealQuery
 
   def index(conn, %{"attendee_id" => attendee_id}) do
     attendee = find_attendee!(attendee_id)
+    meal = find_meal()
+    no_wednesday_meal =
```

Comment:
You have the beginning of a null object here, you can clean it up.

```
meal = find_meal() || %Meal{description: "No Meal"}
```

Then you don't need the conditional, you just pass it straight down the line regardless of what it is. Your template can then decide on how to render it.